packages/cli/src/bin/pipcook-plugin.ts

```diff
@@ -45,7 +48,62 @@ async function list(opts: any): Promise<void> {
     console.table(plugins, [ 'name', 'version', 'category', 'datatype' ]);
   }
 }
+async function installFromConfig(filename: string, opts: any): Promise<void> {
+  const spinner = ora();
+  if (!filename) {
+    spinner.fail('Please specify the config path');
+    return process.exit(1);
+  }
+  let urlObj = url.parse(filename);
+  // file default if the protocol is null
+  if (urlObj.protocol === null) {
+    filename = path.isAbsolute(filename) ? filename : path.join(process.cwd(), filename);
+    // check the filename existence
+    if (!existsSync(filename)) {
+      spinner.fail(`${filename} not exists`);
+      return process.exit(1);
+    } else {
+      filename = url.parse(`file://${filename}`).href;
+    }
+  } else if ([ 'http:', 'https:' ].indexOf(urlObj.protocol) === -1) {
+    spinner.fail(`protocol ${urlObj.protocol} is not supported`);
+    return process.exit(1);
+  }
 
+  const params = {
+    cwd: process.cwd(),
+    config: filename,
+    pyIndex: opts.tuna ? tunaMirrorURI : undefined
+  };
+  if (!opts.verbose) {
+    await get(`${route.plugin}/installFromConfig`, params);
+    spinner.succeed(`install plugins succeeded.`);
```

Comment:
IMO the install command will always be pending.

Reply:
Done.